src/test/standardTest.ts

```diff
@@ -111,6 +111,8 @@ async function start() {
         extensionTestsPath: path.join(EXTENSION_ROOT_DIR_FOR_TESTS, 'out', 'test', 'index'),
         launchArgs: baseLaunchArgs
             .concat([workspacePath])
+            .concat(['--skip-welcome'])
```

Comment:
Also saw these in their test files. Felt like they might be good for test stability so I added them as well.